company_linkage/sql/selecting_top_conference_pubs.sql

```diff
@@ -31,26 +34,28 @@ SELECT
       OR REGEXP_CONTAINS(source_name, r'(?i)\bNeurIPS\b')
     OR REGEXP_CONTAINS(source_name, r'(?i)Annual Meeting of the Association for Computational Linguistics')
       OR ( REGEXP_CONTAINS(source_name, r'\bACL\b')
-      AND conference = 1
+      AND is_conference IS TRUE
     AND NOT REGEXP_CONTAINS(source_name, r'(?i)(injur)|(special interest)|(coling)'))
     OR REGEXP_CONTAINS(source_name, r'(?i)North American Chapter of the Association for Computational Linguistics')
          OR REGEXP_CONTAINS(source_name, r'(?i)\bNAACL\b')
-    OR REGEXP_CONTAINS(source_name, r'(?i)Conference on Empirical Methods in Natural Language Processing')
-      OR REGEXP_CONTAINS(source_name, r'(?i)\bEMNLP\b')
+    OR (REGEXP_CONTAINS(source_name, r'(?i)Conference on Empirical Methods in Natural Language Processing')
+      OR REGEXP_CONTAINS(source_name, r'(?i)\bEMNLP\b'))
+      AND NOT REGEXP_CONTAINS(source_name, r'(?i)SIGDAT')
```

Comment:
Did you intend to have parentheses around lines 41 through 43 so sigdat is only excluded when one of the other two regex match? (I don't know the details of emnlp, so this naive question is only motivated by the way the query is indented!)

Reply:
I did, yeah! Basically it only gets pulled in in those cases so we only need to exclude it in those cases. I suppose broadly excluding it wouldn't be harmful, but I wanted each conference query to be isolated so they could be used in separation as well -- there might be a case when we want, say, ACL results but also something from SIGDAT so I wouldn't want the query as whole to exclude SIGDAT. This may be overthinking, but I have used this query as a foundation for other queries with different paper lists so I like it to be adaptable.

Reply:
Makes sense - I think additional parentheses need to be added in that case